Microsoft.AspNetCore.Components/Microsoft.AspNetCore.Components/IEventCallback.cs

```diff
@@ -0,0 +1,12 @@
+namespace Microsoft.AspNetCore.Components
+{
+	internal interface IEventCallback
+	{
+		bool HasDelegate
+		{
+			get;
+		}
+
+		object? UnpackForRenderTree();
```

Comment:
Is `object?` the correct type here. It appears that in both implementations of this interface the return type is always non-null.